### ISSUE TITLE
"Download PCAPS" right-click is grayed out while "Packets" button is active

### ISSUE THREAD

**Issue by philrz** [brimdata/zui#1572]
In the video by @taosecurity linked from [https://zeek.org/2021/04/14/welcome-to-zeek-in-action-video-1-suspected-malware-compromise/,](https://zeek.org/2021/04/14/welcome-to-zeek-in-action-video-1-suspected-malware-compromise/,) at around the 27:30 mark he hits a snag where he's clicked on a Zeek `http` event and looks to access its pcap slice via the right-click "Download PCAPS" option and finds it grayed out, though the "Packets" button is active. I'm not sure if this is a bug or a technical limitation, and I also expect this may be subject to change in the Brimcap era, but flagging regardless.

**Comment by philrz** [comment]
We went over this one as a group and, as anticipated above, it seems like this is something we'll keep an eye on as part of the work to recreate the pcap-centric workflows with Brimcap. I've marked this one as "blocked" by the relevant Brimcap-related issue and will make sure that it's addressed as part of that, or detached again as its own issue if we come to find it still needs to be handled separately.

**Comment by philrz** [comment]
Verified in Zui commit a9d64be.

I went back and confirmed that this problem was still present in Brim `v0.31.0` but then gone starting with Zui `v1.0.0`. While I didn't narrow down to a specific commit where the change was made, as shown in the attached video, the problem was eliminated by simply dropping the "Download PCAPS" option from the right-click menu altogether. The **Packets** button that's always worked reliably is now the way to extract a pcap slice into Wireshark.

https://github.com/brimdata/zui/assets/5934157/307127d2-8c60-4112-a6fb-a3eba7e43f63
